### PR TITLE
qemu.test: win_heavyload

### PR DESCRIPTION
**Pull request by XueqiangWei**
1. Make it work under both autotest-framework and avocado-framework
2. in migration test, does not stop heavyload on src host

Signed-off-by: Xueqiang Wei <xuwei@redhat.com>

### CODE REVIEW

**Comment by XueqiangWei** [comment]
ID 1342353 1342357 1342366 1342367 1342368

File: qemu/tests/win_heavyload.py [hunk 1]
```diff
@@ -120,16 +124,23 @@ def add_option(cmd, key, val):
         if not loop_session_cmd(session, check_running_cmd):
             raise error.TestError("heavyload process is not started")
 
+        wait_for_migration = int(params.get("wait_for_migration", "0"))
```
Comment (xutian) [inline qemu/tests/win_heavyload.py]: do you mean wait_for_migration done?

Comment (xutian) [inline qemu/tests/win_heavyload.py]: unload_stress_in_the_end   and cleanup_in_the_end  sound more better.

Comment (XueqiangWei) [inline qemu/tests/win_heavyload.py]: no, just wait the heavyload process run some seconds, then do migration

Comment (xutian) [inline qemu/tests/win_heavyload.py]: so it should be sleep_before_migrate right?

Comment (XueqiangWei) [inline qemu/tests/win_heavyload.py]: thx, will update

Comment (XueqiangWei) [inline qemu/tests/win_heavyload.py]: yes, thx

File: qemu/tests/win_heavyload.py [hunk 2]
```diff
@@ -120,16 +124,24 @@ def add_option(cmd, key, val):
...
     finally:
-        error.context("Stop load and clean tmp files", logging.info)
-        if not installed and download_url:
-            utils.system("rm -f %s/HeavyLoad*.exe" % tmp_dir)
-            session.cmd("del /f /s %sHeavyLoad*.exe" % dst)
-        if loop_session_cmd(session, check_running_cmd):
-            if not loop_session_cmd(session, stop_cmd):
-                raise error.TestFail("Unable to terminate heavyload process")
+        # in migration test, no need to stop heavyload on src host
+        cleanup_in_the_end = params.get("unload_stress_in_the_end", "yes")
```
Comment (suqinhuang) [inline qemu/tests/win_heavyload.py]: suggest to update "unload_stress_in_the_end" to "cleanup_in_the_end"

Comment (XueqiangWei) [inline qemu/tests/win_heavyload.py]: updated already, thx

**Comment by PandaWei** [comment]
All comments had been processed. 
ACK.

**Comment by xutian** [comment]
LGTM,  thanks @XueqiangWei and @PandaWei

**Pull request merged**